src/app/users/create-user/create-user.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="app-container">
+    <div class="form-container-parent">
+        <div class="create-user-form-header">
+            <h1>Create User</h1>
+        </div>
+        <div class="form-container">
+
+            <formio [form]='FormData' #formio (submit)="handleSubmit()"></formio>
```

Comment:
Why we need formio here? it  would be cleaner and better to use just normal reactive form.

Reply:
Well, if our users are going to use forms created by formio, why shouldn't we? 😊 I think this will give us consistency as there will be lot of forms

Reply:
Also, using forms created by formio will give us some more understanding about its usability.